<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="location_marker_a">
    <static>true</static>
    <link name="link">
      <pose>0 0 1 0 0 0</pose>
      <collision name="collision">
        <geometry>
          <box>
            <size>1 1 2</size>
          </box>
        </geometry>
      </collision>
      <visual name="foot">
        <pose>0 0 -1.5 0 0 0</pose>
        <geometry>
          <box>
            <size>1 1 1</size>
          </box>
        </geometry>
      </visual>
      <visual name="visual_bottom">
        <pose>0 0 -0.75 0 0 0</pose>
        <geometry>
          <box>
            <size>1 1 0.5</size>
          </box>
        </geometry>
        <material>
          <script>
            <uri>model://location_marker_a/materials/scripts</uri>
            <uri>model://location_marker_a/materials/textures</uri>
            <name>location_marker_a/logo</name>
          </script>
          <pbr>
            <metal>
              <albedo_map>model://location_marker_a/materials/textures/fre_logo.png</albedo_map>
            </metal>
          </pbr>
        </material>
      </visual>
      <visual name="visual_top">
        <pose>0 0 0.75 0 0 0</pose>
        <geometry>
          <box>
            <size>1 1 0.5</size>
          </box>
        </geometry>
        <material>
          <script>
            <uri>model://location_marker_a/materials/scripts</uri>
            <uri>model://location_marker_a/materials/textures</uri>
            <name>location_marker_a/logo</name>
          </script>
          <pbr>
            <metal>
              <albedo_map>model://location_marker_a/materials/textures/fre_logo.png</albedo_map>
            </metal>
          </pbr>
        </material>
      </visual>
      <visual name="visual_code">
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <box>
            <size>1 1 1</size>
          </box>
        </geometry>
        <material>
          <script>
            <uri>model://location_marker_a/materials/scripts</uri>
            <uri>model://location_marker_a/materials/textures</uri>
            <name>location_marker_a/qr_code</name>
          </script>
          <pbr>
            <metal>
              <albedo_map>model://location_marker_a/materials/textures/location_marker_a.png</albedo_map>
            </metal>
          </pbr>
        </material>
      </visual>
    </link>
  </model>
</sdf>
